projects/1.20-fabric/better-archeology/betterarcheology/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+  "block.betterarcheology.cracked_mud_bricks": "裂纹泥砖",
+  "block.betterarcheology.cracked_mud_brick_slab": "裂纹泥砖台阶",
+  "block.betterarcheology.cracked_mud_brick_stairs": "裂纹泥砖楼梯",
+  "block.betterarcheology.suspicious_red_sand": "可疑的红沙",
+  "block.betterarcheology.suspicious_dirt": "可疑的泥土",
+  "block.betterarcheology.fossiliferous_dirt": "含化石的泥土",
+  "block.betterarcheology.fossiliferous_dirt_tooltip": "也许其中有什么东西等待着发掘？",
+  "block.betterarcheology.villager_fossil": "村民化石",
+  "block.betterarcheology.villager_fossil_tooltip": "它变为化石的手臂或许可以展示什么东西……",
+  "block.betterarcheology.villager_fossil_head": "村民颅骨化石",
+  "block.betterarcheology.villager_fossil_head_tooltip": "他们看起来还没开始进化……",
+  "block.betterarcheology.villager_fossil_body": "村民躯干化石",
+  "block.betterarcheology.villager_fossil_body_tooltip": "这个可怖的东西提醒着人们曾有一座村庄消失在时间之中。",
+  "block.betterarcheology.ocelot_fossil": "豹猫化石",
+  "block.betterarcheology.ocelot_fossil_tooltip": "苦力怕们怕这个更甚于怕活猫。",
+  "block.betterarcheology.ocelot_fossil_head": "豹猫颅骨化石",
+  "block.betterarcheology.ocelot_fossil_head_tooltip": "不同于它们的后裔，这群豹猫有着长长的利齿。",
+  "block.betterarcheology.ocelot_fossil_body": "豹猫躯干化石",
+  "block.betterarcheology.ocelot_fossil_body_tooltip": "看起来它们牺牲敏捷换来了力量。",
+  "block.betterarcheology.sheep_fossil": "羊化石",
+  "block.betterarcheology.sheep_fossil_tooltip": "它的角让你想起了山羊的。也许这个也能吹得响？",
+  "block.betterarcheology.sheep_fossil_head": "羊颅骨化石",
+  "block.betterarcheology.sheep_fossil_head_tooltip": "也许它能撞出比山羊更深刻的凹痕。",
+  "block.betterarcheology.sheep_fossil_body": "羊躯干化石",
+  "block.betterarcheology.sheep_fossil_body_tooltip": "这群羊蓬松松的毛没了。",
+  "block.betterarcheology.chicken_fossil": "鸡化石",
+  "block.betterarcheology.chicken_fossil_tooltip": "靠近它的时候你感觉身体变轻了。",
+  "block.betterarcheology.chicken_fossil_head": "鸡颅骨化石",
+  "block.betterarcheology.chicken_fossil_head_tooltip": "它的喙比你熟知的鸡更大。",
+  "block.betterarcheology.chicken_fossil_body": "鸡躯干化石",
+  "block.betterarcheology.chicken_fossil_body_tooltip": "它的翅膀上有更尖利的爪子；它们过去可能也有着更长的尾巴。",
+  "block.betterarcheology.creeper_fossil": "苦力怕化石",
+  "block.betterarcheology.creeper_fossil_tooltip": "它比真的苦力怕看着还吓人……",
+  "block.betterarcheology.creeper_fossil_head": "苦力怕颅骨化石",
+  "block.betterarcheology.creeper_fossil_head_tooltip": "这副脸曾让你陷入无休止的战栗之中。",
+  "block.betterarcheology.creeper_fossil_body": "苦力怕躯干化石",
+  "block.betterarcheology.creeper_fossil_body_tooltip": "所以这就是它力量的源头！",
+  "block.betterarcheology.loot_vase": "古代花瓶",
+  "block.betterarcheology.vase": "古代花瓶",
+  "block.betterarcheology.loot_vase_creeper": "古代花瓶",
+  "block.betterarcheology.vase_creeper": "古代花瓶",
+  "block.betterarcheology.evoker_trap": "唤魔者陷阱",
+
+  "advancements.shards": "破碎的知识",
+  "advancements.shards.description": "找到文物碎片。",
+  "advancements.artifact": "重聚",
+  "advancements.artifact.description": "使用文物碎片合成未经鉴定的文物，现在开始鉴定吧！",
+  "advancements.vase": "成功敲开它",
+  "advancements.vase.description": "打碎花瓶以获得战利品。",
+  "advancements.fossiliferous_dirt": "“挖”越时间",

```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**Digging through Time**
```suggestion
  "advancements.fossiliferous_dirt": "掘今追昔",
```
瞎整的，瞎整的

---

这个好，就这个了